cars/car567.md

```diff
@@ -0,0 +1 @@
+This is now car567.md
```

Comment:
Piotr - would you please add additional information here?

Reply:
VIN number, make, model, etc.

Reply:
More details added